iidm/iidm-mergingview/src/main/java/com/powsybl/iidm/mergingview/MergingView.java

```diff
@@ -220,41 +222,62 @@ public boolean hasProperty() {
     }
 
     @Override
-    public boolean hasProperty(final String key) {
+    public boolean hasProperty(String key) {
         return index.getNetworkStream()
                 .anyMatch(n -> n.hasProperty(key));
     }
 
     @Override
-    public String getProperty(final String key) {
+    public Properties.Type getPropertyType(String key) {
         return index.getNetworkStream()
-                .map(n -> n.getProperty(key))
-                .filter(Objects::nonNull)
-                .findFirst()
-                .orElse(null);
+            .map(n -> n.getPropertyType(key))
+            .filter(Objects::nonNull)
+            .findFirst()
+            .orElse(null);
```

Comment:
Configure your IDE with 8 spaces to continue indentation and revert all the unnecessary changes due to your bad configuration. This remark is general for all this PR.